README.md

```diff
@@ -1170,6 +1170,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
    * [bit.io](https://bit.io) — Managed PostgreSQL database service. 3 database, 3GB storage per database, 1 CPU and 1GB Memory (burst), 1B rows queried per month
    * [codehooks.io](https://codehooks.io/) — JavaScript serverless API/backend and database service with functions, Mongdb-ish queries, key/value lookups, a job system and a message queue. 1 instance free per project, 5000 records and 5000 calls/month free, 3 developers included. No credit-card required.
    * [CrateDB](https://crate.io/) - Distributed Open Source SQL database for real-time analytics. [Free Tier CRFREE](https://crate.io/lp-crfree): One-node with 2 CPUs, 2 GiB of memory, 8 GiB of storage. 1 cluster per organization, no payment method needed.
+   * [DBHub.io](https://dbhub.io) - SQLite database hosting, fully Open Source platform, from the creators of [DB Browser for SQLite](https://sqlitebrowser.org). Public databases are free.
```

Comment:
Seems like a strange thing to say.

DBHub.io has been completely free for the past 5+ years, and part you're quoting says we're intending on keeping public databases be free.

The contribution note at the top of your doc says:

```
If the free tier is time-bucketed it has to be for at least a year.
```

So I'm not sure why you're suddenly raising the bar to being "free forever"???

Reply:
It says “initially everything will be free” does initially mean a free tier for a year? Does it mean there will forever be a free tier? Doesn’t seem to say that. 

Maybe I missed it, maybe you can quote the passage saying you intend to keep having a free tier. 

The rule isn’t it must be free for ever - it can be time bucketed to free for a year. But this is a list for services with free tiers so we need to see it clearly that there is a free tier and not just an experiment till something else is figured out.

Reply:
Well, "initially" has been over 5 years so far. :smile:

Reply:
And to re-iterate, for *public* databases our intention is to keep them free regardless.

Reply:
Cool, so I must have missed that undertaking on the page please let me know where to see that or share a screenshot so we can get it merged.

Reply:
Just double checked.  I was wrong about it being 5 years so far.  Sorry about that.

It's been nearly 7 years. :smile:

Reply:
Not sure what your aim is here. 

100s of people maintain this list. When they visit a site and cannot see the free claims clearly marked on the site they edit the entry to remove it. 

We have 2 options to avoid that. We can clearly state the terms on the site or we can spare everyone some effort and not add it. 

So if the site clearly states the offering we can add it otherwise, and if you don’t want to clarify it, please close the PR

Things like how long it has been free or whatever doesn’t help. The information needs to be clearly visible.

Reply:
Ok, I'm not understanding something.

Is it not clear that we're providing free hosting for public databases, that we have been doing so for many years, and we intent to keep on doing so?

Reply:
Hmmm, I think I see where you're coming from.  We need to update the wording on our About page to make things super clear.

Ok, that's doable.